examples-native/crna-kitchen-sink/README.md

```diff
@@ -1,3 +1,9 @@
 # CRNA Kitchen Sink
 
-This project was bootstrapped with [Create React Native App](https://github.com/react-community/create-react-native-app) and storybook using [storybook CLI](https://www.npmjs.com/package/@storybook/cli).
```

Comment:
Maybe just change the sentence that it was bootstrapped with expo?

Reply:
done 👍